src/db_selection.ts

```diff
@@ -0,0 +1,42 @@
+// TODO: We should probably use an interface here
```

Comment:
I wanted to use an interface here but I want to run the idea by you first. This list of if statements should work fine for the time being.